src/mon/Monitor.cc

```diff
@@ -3445,6 +3445,7 @@ void Monitor::handle_command(MonOpRequestRef op)
   }
   catch(bad_cmd_get& e) {
```

Comment:
I'd also fix the formatting and missing const while at it:
```suggestion
  } catch (const bad_cmd_get& e) {
```

Reply:
Hi @idryomov , thanks!

I created https://tracker.ceph.com/issues/57859 for this, and re-submitted the MR. 

Please may I request you to link this pull request in it, I am not able to modify the description once I created the issue.

Regards,
Nikhil.

Reply:
> Please may I request you to link this pull request in it, I am not able to modify the description once I created the issue.

Done